server/cluster/cluster.go

```diff
@@ -1207,6 +1239,28 @@ func (c *RaftCluster) checkStores() {
 	}
 }
 
+func (c *RaftCluster) updateProgress(storeID uint64, storeAddress string, action string, current int64) {
+	storeLabel := fmt.Sprintf("%d", storeID)
+	progress := fmt.Sprintf("%s-%s", action, storeLabel)
+	if !c.progressManager.IsProgressExist(progress) {
+		c.progressManager.AddProgressIndicator(progress, float64(current))
```

Comment:
Should we consider the following situation(t1<t2<t3<t4)?
t1: thread 1 has finished L1245 and gets a `false` from `IsProgressExist`
t2: thread 2  share the same `progress` with thread 1 and gets a `false` from `IsProgressExist`
t3: thread 1 finish L1246 `AddProgressIndicator`, and now the value in `progressManager` was initted by current1
t4: thread 2 finish L1246 `AddProgressIndicator`, and now the value in `progressManager` was replaced by current2

Reply:
ditto